storage/src/lib.rs

```diff
@@ -118,14 +118,50 @@ pub enum TrieNodeParent {
 }
 
 /// This enum encapsulates what points to the stored area allocated for a free list.
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
 pub enum FreeListParent {
     /// The stored area is the head of the free list, so the header points to it
     FreeListHead(AreaIndex),
     /// The stored area is not the head of the free list, so a previous free area points to it
     PrevFreeArea(LinearAddress),
 }
 
+impl std::fmt::LowerHex for StoredAreaParent {
```

Comment:
`use std::fmt` here, then use `impl fmt::LowerHex`. This removes a lot of `std::fmt` stutter below too.